2020-08-05T01:13:32.3892312+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T01:13:32.5140748+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T01:13:32.5142251+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T01:13:47.9534221+05:30 8000002b-0001-f400-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 01:13:47  (8151f914)
2020-08-05T01:19:57.9851154+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T01:19:58.0077503+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T01:19:58.0079197+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T01:20:09.8576243+05:30 80000038-0002-fc00-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 01:20:09  (8151f914)
2020-08-05T01:21:20.6909600+05:30 8000003e-0002-fa00-b63f-84710c7967bb [ERR] Failed executing DbCommand ("37"ms) [Parameters=["@MEMBER_ID='?'"], CommandType='Text', CommandTimeout='30']"
""EXEC [dbo].[MOS_ASSESSMENT_HISTORY_READ] @MEMBER_ID" (627a98df)
2020-08-05T01:21:20.6909596+05:30 8000003e-0002-f900-b63f-84710c7967bb [ERR] Failed executing DbCommand ("233"ms) [Parameters=["@MEMBER_ID='?'"], CommandType='Text', CommandTimeout='30']"
""EXEC [dbo].[MOS_ASSESSMENT_HISTORY_READ] @MEMBER_ID" (627a98df)
2020-08-05T01:21:20.7818024+05:30 8000003e-0002-fa00-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""Microsoft.Data.SqlClient.SqlException (0x80131904): The parameterized query '(@MEMBER_ID nvarchar(4000))EXEC [dbo].[MOS_ASSESSMENT_HISTORY_RE' expects the parameter '@MEMBER_ID', which was not supplied.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:5527be35-82fc-4149-85b0-b4373adca903
Error Number:8178,State:1,Class:16" (f5c7ac34)
Microsoft.Data.SqlClient.SqlException (0x80131904): The parameterized query '(@MEMBER_ID nvarchar(4000))EXEC [dbo].[MOS_ASSESSMENT_HISTORY_RE' expects the parameter '@MEMBER_ID', which was not supplied.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:5527be35-82fc-4149-85b0-b4373adca903
Error Number:8178,State:1,Class:16
2020-08-05T01:21:20.7828190+05:30 8000003e-0002-f900-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""Microsoft.Data.SqlClient.SqlException (0x80131904): The parameterized query '(@MEMBER_ID nvarchar(4000))EXEC [dbo].[MOS_ASSESSMENT_HISTORY_RE' expects the parameter '@MEMBER_ID', which was not supplied.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
ClientConnectionId:e1626c2b-e440-4c8b-98fc-55c566a52647
Error Number:8178,State:1,Class:16" (f5c7ac34)
Microsoft.Data.SqlClient.SqlException (0x80131904): The parameterized query '(@MEMBER_ID nvarchar(4000))EXEC [dbo].[MOS_ASSESSMENT_HISTORY_RE' expects the parameter '@MEMBER_ID', which was not supplied.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 34
ClientConnectionId:e1626c2b-e440-4c8b-98fc-55c566a52647
Error Number:8178,State:1,Class:16
2020-08-05T01:21:21.8017708+05:30 8000003e-0002-fa00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
Microsoft.Data.SqlClient.SqlException (0x80131904): The parameterized query '(@MEMBER_ID nvarchar(4000))EXEC [dbo].[MOS_ASSESSMENT_HISTORY_RE' expects the parameter '@MEMBER_ID', which was not supplied.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
ClientConnectionId:5527be35-82fc-4149-85b0-b4373adca903
Error Number:8178,State:1,Class:16
2020-08-05T01:21:21.8017717+05:30 8000003e-0002-f900-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
Microsoft.Data.SqlClient.SqlException (0x80131904): The parameterized query '(@MEMBER_ID nvarchar(4000))EXEC [dbo].[MOS_ASSESSMENT_HISTORY_RE' expects the parameter '@MEMBER_ID', which was not supplied.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
ClientConnectionId:e1626c2b-e440-4c8b-98fc-55c566a52647
Error Number:8178,State:1,Class:16
2020-08-05T01:58:33.0304767+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T01:58:33.1161943+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T01:58:33.1163437+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T01:58:51.5401848+05:30 8000001d-0000-f800-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 01:58:51  (8151f914)
2020-08-05T02:02:17.5187412+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:02:17.6177484+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:02:17.6179295+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:02:35.8442289+05:30 80000043-0002-fa00-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:02:35  (8151f914)
2020-08-05T02:06:10.3409517+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:06:10.3844696+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:06:10.3846055+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:06:24.9139375+05:30 80000043-0002-f900-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:06:24  (8151f914)
2020-08-05T02:09:13.2518751+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:09:13.2716546+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:09:13.2717993+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:09:24.2601170+05:30 80000019-0002-f500-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:09:24  (8151f914)
2020-08-05T02:15:04.8294608+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:15:04.8443333+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:15:04.8445367+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:15:16.5626174+05:30 8000002d-0003-f500-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:15:16  (8151f914)
2020-08-05T02:20:56.5390562+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:20:56.5651102+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:20:56.5652414+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:21:07.6918211+05:30 80000030-0003-f100-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:21:07  (8151f914)
2020-08-05T02:24:55.3122897+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:24:55.3455472+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:24:55.3456691+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:25:23.2957706+05:30 8000005a-0003-f800-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:25:23  (8151f914)
2020-08-05T02:38:13.9916604+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:38:14.0167952+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:38:14.0169962+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:38:24.7192658+05:30 80000039-0001-f100-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:38:24  (8151f914)
2020-08-05T02:39:14.5576736+05:30 8000006a-0002-f700-b63f-84710c7967bb [ERR] Failed executing DbCommand ("11"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""SELECT [a].[Sgk_Mbr_Assessment_Id], [a].[Assessment_ID], [a].[Disease_Type_ID], [a].[Insert_DTS], [a].[Insert_Usr_Id], [a].[Member_Id], [a].[Notes], [a].[Update_DTS], [a].[Update_Usr_Id], [a].[sgk_Option_Map_ID]
FROM [assessments] AS [a]" (627a98df)
2020-08-05T02:39:14.6332022+05:30 8000006a-0002-f700-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid object name 'assessments'.
   at Microsoft.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at Microsoft.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at Microsoft.Data.SqlClient.SqlDataReader.get_MetaData()
   at Microsoft.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean isAsync, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry, String method)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at Microsoft.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at Microsoft.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
ClientConnectionId:ac2a0b13-0cdb-4ed8-b6b4-6043209c16be
Error Number:208,State:1,Class:16" (f5c7ac34)
Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid object name 'assessments'.
   at Microsoft.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at Microsoft.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at Microsoft.Data.SqlClient.SqlDataReader.get_MetaData()
   at Microsoft.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean isAsync, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry, String method)
   at Microsoft.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at Microsoft.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at Microsoft.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.InitializeReader(DbContext _, Boolean result)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.Enumerator.MoveNext()
   at System.Collections.Generic.LargeArrayBuilder`1.AddRange(IEnumerable`1 items)
   at System.Collections.Generic.EnumerableHelpers.ToArray[T](IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at System.Linq.SystemCore_EnumerableDebugView`1.get_Items()
ClientConnectionId:ac2a0b13-0cdb-4ed8-b6b4-6043209c16be
Error Number:208,State:1,Class:16
2020-08-05T02:39:33.0909340+05:30 8000006a-0002-f700-b63f-84710c7967bb [ERR] Failed executing DbCommand ("3"ms) [Parameters=["@p0='?' (DbType = Int32), @p1='?' (DbType = Int32), @p2='?' (DbType = DateTime2), @p3='?' (DbType = Int32), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (DbType = DateTime2), @p7='?' (DbType = Int32), @p8='?' (DbType = Int32)"], CommandType='Text', CommandTimeout='30']"
""SET NOCOUNT ON;
INSERT INTO [assessments] ([Assessment_ID], [Disease_Type_ID], [Insert_DTS], [Insert_Usr_Id], [Member_Id], [Notes], [Update_DTS], [Update_Usr_Id], [sgk_Option_Map_ID])
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, @p7, @p8);
SELECT [Sgk_Mbr_Assessment_Id]
FROM [assessments]
WHERE @@ROWCOUNT = 1 AND [Sgk_Mbr_Assessment_Id] = scope_identity();" (627a98df)
2020-08-05T02:42:26.7986168+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:42:26.8172229+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:42:26.8176065+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:42:38.5898461+05:30 8000001c-0001-fb00-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:42:38  (8151f914)
2020-08-05T02:47:02.7319890+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:47:02.7519430+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:47:02.7520924+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:47:24.1464924+05:30 80000045-0002-f600-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:47:24  (8151f914)
2020-08-05T02:47:47.5297575+05:30 80000046-0002-f600-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:47:47  (8151f914)
2020-08-05T02:51:56.0651096+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:51:56.1353437+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:51:56.1355387+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:52:09.4466967+05:30 8000004a-0001-f700-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:52:09  (8151f914)
2020-08-05T02:57:40.3687071+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T02:57:40.4207808+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T02:57:40.4209651+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T02:57:53.7850311+05:30 8000004c-0001-f700-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 02:57:53  (8151f914)
2020-08-05T03:00:29.7087634+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:00:29.7756295+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:00:29.7758171+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:00:42.1504542+05:30 8000003f-0001-f400-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:00:42  (8151f914)
2020-08-05T03:05:19.3306911+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:05:19.4716509+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:05:19.4720963+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:05:32.7136576+05:30 8000005b-0002-f900-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:05:32  (8151f914)
2020-08-05T03:07:47.2006089+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:07:47.2355270+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:07:47.2356951+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:08:00.1586644+05:30 80000064-0001-f500-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:08:00  (8151f914)
2020-08-05T03:08:11.3092329+05:30 80000065-0001-f500-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:08:11  (8151f914)
2020-08-05T03:32:30.9954260+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:32:31.0613812+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:32:31.0619100+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:32:45.3605154+05:30 80000062-0002-fb00-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:32:45  (8151f914)
2020-08-05T03:33:21.3404579+05:30 80000052-0001-f700-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:33:21.7138830+05:30 80000052-0001-f700-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:35:27.9081510+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:35:27.9424530+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:35:27.9425940+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:35:46.2234384+05:30 80000036-0003-f100-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:35:46  (8151f914)
2020-08-05T03:36:26.9203010+05:30 80000043-0003-f500-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:36:27.3334613+05:30 80000043-0003-f500-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:37:00.1494644+05:30 80000044-0003-f500-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:37:00.4043540+05:30 80000044-0003-f500-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:39:51.2626798+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:39:51.3311466+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:39:51.3315734+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:40:03.1432590+05:30 80000070-0002-fb00-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:40:03  (8151f914)
2020-08-05T03:41:00.7131588+05:30 80000040-0001-f600-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:41:04.5253778+05:30 80000040-0001-f600-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:41:32.3310680+05:30 80000041-0001-f600-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:42:48.4473530+05:30 80000041-0001-f600-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Disease' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:47:16.0392480+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:47:16.0616692+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:47:16.0624908+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:49:40.3217849+05:30 80000046-0001-f600-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:49:40  (8151f914)
2020-08-05T03:52:43.1429679+05:30 80000082-0002-fb00-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:52:43.5474962+05:30 80000082-0002-fb00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:53:38.3709166+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:53:38.3895901+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:53:38.3898029+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:53:59.2139642+05:30 80000015-0000-f300-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:53:59  (8151f914)
2020-08-05T03:54:42.1647979+05:30 80000030-0001-fb00-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:54:42.5096077+05:30 80000030-0001-fb00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:54:43.1224365+05:30 80000031-0001-fb00-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:54:43.3910345+05:30 80000031-0001-fb00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:55:01.3178890+05:30 80000032-0001-fb00-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:55:01.6127428+05:30 80000032-0001-fb00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:55:01.9628577+05:30 80000033-0001-fb00-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:55:02.2302847+05:30 80000033-0001-fb00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:56:08.8964690+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T03:56:08.9134658+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T03:56:08.9141100+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T03:56:19.7375874+05:30 80000057-0001-f700-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 03:56:19  (8151f914)
2020-08-05T03:56:55.0582037+05:30 80000020-0001-f200-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:56:55.4466502+05:30 80000020-0001-f200-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:56:55.9883726+05:30 80000021-0001-f200-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:56:56.3330116+05:30 80000021-0001-f200-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:57:47.7730081+05:30 80000022-0001-f200-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:57:48.0306083+05:30 80000022-0001-f200-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:58:31.7187883+05:30 80000023-0001-f200-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:58:31.9751271+05:30 80000023-0001-f200-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T03:59:20.6492862+05:30 80000024-0001-f200-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T03:59:20.9197209+05:30 80000024-0001-f200-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T04:00:23.9966196+05:30 80000025-0001-f200-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T04:00:24.3659103+05:30 80000025-0001-f200-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T04:01:25.6091334+05:30 80000026-0001-f200-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T04:01:26.0006952+05:30 80000026-0001-f200-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at MemberOutReachSystem.Business.AssessmentRepository.GetAssessmentDetails(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.Business\AssessmentRepository.cs:line 38
   at MemberOutReachSystem.API.Controllers.AssessmentController.Get(String MemberId) in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\AssessmentController.cs:line 33
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T04:01:44.3581039+05:30 80000027-0001-f200-b63f-84710c7967bb [ERR] An exception occurred while iterating over the results of a query for context type '"MemberOutReachSystem.Data.OutreachDbContext"'."
""System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()" (f5c7ac34)
System.InvalidOperationException: The required column 'Insert_DTS' was not present in the results of a 'FromSql' operation.
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.BuildIndexMap(IReadOnlyList`1 columnNames, DbDataReader dataReader)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
2020-08-05T04:02:44.3831932+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T04:02:44.4354803+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T04:02:44.4499375+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T04:03:05.4261751+05:30 80000036-0002-f800-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 04:03:05  (8151f914)
2020-08-05T11:56:32.9036458+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T11:56:33.0344939+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T11:56:33.0346386+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T11:56:48.5833367+05:30 80000058-0000-f900-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 11:56:48  (8151f914)
2020-08-05T16:11:26.1573796+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:11:26.1828881+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:11:26.1833252+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:20:55.8501631+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:20:55.8745136+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:20:55.8749423+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:21:20.6067892+05:30 80000025-0000-f300-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 16:21:20  (8151f914)
2020-08-05T16:22:43.2120946+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:22:43.3744644+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:22:43.3747469+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:22:59.1866790+05:30 80000012-0001-f000-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 16:22:59  (8151f914)
2020-08-05T16:25:30.7938997+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:25:30.8176564+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:25:30.8178152+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:25:42.2930177+05:30 80000047-0002-f500-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 16:25:42  (8151f914)
2020-08-05T16:41:49.9173416+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:41:49.9304621+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:41:49.9306287+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:42:02.0742196+05:30 8000006f-0002-f900-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 16:42:02  (8151f914)
2020-08-05T16:44:40.2816324+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:44:40.3272399+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:44:40.3274265+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:44:53.5832297+05:30 80000071-0003-f200-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 16:44:53  (8151f914)
2020-08-05T16:48:01.4576174+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:48:01.5950785+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:48:01.5954741+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:48:15.1357621+05:30 80000090-0003-f800-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 16:48:15  (8151f914)
2020-08-05T16:49:55.0292572+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:49:55.0517484+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:49:55.0523461+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:50:14.2690589+05:30 80000051-0003-f300-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 16:50:14  (8151f914)
2020-08-05T16:56:58.1961526+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T16:56:58.2120754+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T16:56:58.2126028+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T16:57:17.7810003+05:30 80000040-0000-f700-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 16:57:17  (8151f914)
2020-08-05T17:03:51.0705659+05:30 8000006d-0001-f900-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 17:03:51  (8151f914)
2020-08-05T17:38:35.0385611+05:30 8000005f-0003-f500-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 17:38:35  (8151f914)
2020-08-05T18:19:06.8793977+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T18:19:06.9072136+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T18:19:06.9078447+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T18:19:51.3023014+05:30 80000099-0000-fd00-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 18:19:51  (8151f914)
2020-08-05T18:35:26.0990480+05:30 80000052-0000-f800-b63f-84710c7967bb [INF] Validate User Request on Date:08/05/2020 18:35:26  (8151f914)
2020-08-05T18:42:41.9518130+05:30  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-08-05T18:42:42.0206965+05:30  [INF] Hosting environment: "Development" (c3307c92)
2020-08-05T18:42:42.0209780+05:30  [INF] Content root path: "D:\data\C_drive\source\repos\MemberOutReachSystem.API" (b5d60022)
2020-08-05T18:43:01.8384680+05:30 800000ac-0003-f600-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.Exception: Error in member Service
   at MemberOutReachSystem.API.Controllers.MemberController.Get() in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\MemberController.cs:line 31
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2020-08-05T18:43:18.1418521+05:30 800000ad-0003-f600-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.Exception: Error in member Service
   at MemberOutReachSystem.API.Controllers.MemberController.Get() in D:\data\C_drive\source\repos\MemberOutReachSystem.API\Controllers\MemberController.cs:line 31
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
